spyderlib/spyder.py

```diff
@@ -2646,6 +2646,9 @@ def start_open_files_server(self):
                 # To avoid a traceback after closing on Windows
                 if e.args[0] == eintr:
                     continue
+                # handle a connection abort on close error
+                if e.args[0] in [errno.ECONNABORTED, errno.ENOTSOCK]:
+                    return                

```

Comment:
Why here `return` and not `continue`? I say it because I tried to fix this error in commit e18294fbba9a91c815a234da8df16b18824e2bb0 but I thought the right thing to do was to use `continue`. Later I had to revert that one because complained that Spyder took up to half an hour to start or close!! :-)